draft-ietf-quic-recovery.md

```diff
@@ -1792,6 +1789,24 @@ OnPacketsLost(lost_packets):
     congestion_recovery_start_time = 0
 ~~~
 
+
+## Removing Discarded Packets From Bytes In Flight
+
+When Initial or Handshake keys are discarded, packets sent in that space no
+longer count toward bytes in flight.
+
+Pseudocode for RemoveFromBytesInFlight follows:
+
+~~~
+RemoveFromBytesInFlight(discarded):
+  // Remove any unacknowledged packets from flight.
+  foreach packet in discarded:
```

Comment:
```suggestion
RemoveFromBytesInFlight(discarded_pn_space):
  // Remove any unacknowledged packets from flight.
  foreach packet in discarded_pn_space:
```

Reply:
Would "discarded_packets" be better?  It's really just "discarded packets" which only happen to be from a given packet number space.

Reply:
Yes, after re-reading the text, discarded_packets seems correct.